hs_core/views/__init__.py

```diff
@@ -58,6 +61,24 @@ def verify(request, *args, **kwargs):
             u.is_active=True
             u.save()
             u.groups.add(Group.objects.get(name="Hydroshare Author"))
+
+        # register the user in HSAccess db accordingly
+        try:
+            ha_obj = HSAlib.HSAccess('admin', 'unused', django_settings.HS_ACCESS_DB, django_settings.HS_ACCESS_USERNAME,
+                                     django_settings.HS_ACCESS_PASSWORD, django_settings.HS_ACCESS_HOST, django_settings.HS_ACCESS_PORT)
+        except:
+            # unable to connect to the database
+            raise DatabaseError("unable to connect to the database HSAccess.")
+
+        try:
+            ha_obj.assert_user(str(u.username), 'HydroShare User')
+        except:
+            # unable to connect to the database
+            raise OperationalError("login %s cannot be inserted into HSAccess database." % u.username)
```

Comment:
That is an interesting idea and we can discuss today in our dev call if you can join us. I am not sure whether we need to add new users to HSAccess periodically, though, since I think if it fails in the first time, it will always fail, or do I miss something. Anyway, we can discuss in detail later.